linux_pipeline/Jenkinsfile_azure_lis_hotfix_functional

```diff
@@ -0,0 +1,344 @@
+#!/usr/bin/env groovy
+def currentStage = ""
+def armImagesCentOS = ["rhel_6.5": "OpenLogic CentOS 6.5 Latest", "rhel_6.6": "OpenLogic CentOS 6.6 Latest", "rhel_6.7": "OpenLogic CentOS 6.7 Latest", 
```

Comment:
@iamshital you and I discussed about the similar situation this morning. "hard-corded" distro name and test cases are not the good idea. Please consult to @prashantmhetar for better approach.

Reply:
@juhlee-microsoft This is just variable to actual value mapping. The variables (rhel_6.5 / rhel_6.6 etc) are passed as parameters.

Reply:
@prashantmhetar Let's update to run P0 tests in first group and Additional tests in second group.
@juhlee-microsoft , We've created a test matrix as per the LIS release test plan, hence, we're executing the tests that are relevant to LIS release.

Reply:
Updated the Functional test pipeline in 2 different groups